apps/web-giddh/src/app/theme/ngx-date-range-picker/ngx-daterangepicker.component.ts

```diff
@@ -1660,29 +1678,47 @@ export class NgxDaterangepickerComponent implements OnInit, OnDestroy {
     public getFinancialYears(): void {
         this.settingsFinancialYearService.GetAllFinancialYears().subscribe(res => {
             if (res && res.body && res.body.financialYears) {
```

Comment:
Consider checking if financialYears length > 0 as 0th index is accessed later in code
`moment(moment(res.body.financialYears[0].financialYearStarts, GIDDH_DATE_FORMAT).toDate())`